lib/template/macros.c

```diff
@@ -570,6 +570,8 @@ log_macro_expand(GString *result, gint id, gboolean escape, const LogTemplateOpt
           length = format_zone_info(buf, sizeof(buf), zone_ofs);
           g_string_append_len(result, buf, length);
           break;
+        default:
```

Comment:
This should cover all macro types. The default branch should indicate a bug. Please add g_assert_not_reached() here.